android/app/src/main/res/values/color.xml

```diff
@@ -9,6 +9,7 @@
     <color name="primary_text">#212121</color>
     <color name="secondary_text">#727272</color>
     <color name="translucent_grey">#0c000000</color>
+    <color name="speaker_name_search_color">#1565C0</color>
```

Comment:
Use theme colors instead, we just develop template app. Hard coding opinionated colors is bad choice